ovirtlago/virt.py

```diff
@@ -300,18 +321,12 @@ def _host_is_up():
                     )
                 elif host_obj.status == sdk4.types.HostStatus.INSTALL_FAILED:
                     raise RuntimeError('Host %s installation failed' % h.name)
-                elif host_obj.status == sdk4.types.HostStatus.NON_RESPONSIVE:
```

Comment:
Its not an exception :/ For the good and bad:
```
In [3]: from ovirtsdk4 import types

In [4]: type(types.HostStatus)
Out[4]: enum.EnumMeta

In [5]: type(types.HostStatus.NON_RESPONSIVE)
Out[5]: <enum 'HostStatus'>
```

There is some refactoring we can do here, with https://github.com/lago-project/lago/pull/483 or without, but I think we can do it later.